pwndbg/gdblib/symbol.py

```diff
@@ -141,13 +140,13 @@ def address(symbol: str) -> int:
 
     try:
         # Unfortunately, `gdb.lookup_symbol` does not seem to handle all
-        # symbols, so we need to fallback to using `info address`. See
+        # symbols, so we need to fallback to using `gdb.parse_and_eval`. See
         # https://sourceware.org/pipermail/gdb/2022-October/050362.html
-        address = pwndbg.gdblib.info.address(symbol)
-        if address is None or not pwndbg.gdblib.vmmap.find(address):
-            return None
-
-        return address
+        # (We tried parsing the output of the `info symbol` before, but there were some issues. See #1628 and #1666)
+        if "'" in symbol:
+            # Is it possible that happens? Probably not, but just in case
```

Comment:
Maybe in Rust with lifetime specifiers?

Reply:
Oh, cool, I didn't know that.
How about check `\` instead, and sanitize all `'` to `\'`?
Is it possible that we need to use `\` in the symbol name?
Or any possibility that some input can escape from this check?

Reply:
I dont think symbols use `\` and we can always fix it if a bug with this appears... ;D